<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Add</class>
 <widget class="QDialog" name="Add">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>546</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0.903409, stop:0 rgba(18, 22, 29, 255), stop:1 rgba(55, 59, 72, 255));
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font/>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
color:white;
}</string>
   </property>
   <property name="text">
    <string>New Quote</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font/>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
color:white;
}</string>
   </property>
   <property name="text">
    <string>Author</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>180</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font/>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
color:white;
}</string>
   </property>
   <property name="text">
    <string>Category</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_author">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(72, 72, 72, 255), stop:1 rgba(90, 90, 90, 255));
border-style: offset;
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_cgy">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>220</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(72, 72, 72, 255), stop:1 rgba(90, 90, 90, 255));
border-style: offset;
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>340</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_data">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>340</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: white;
color:black;
}</string>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_quote">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>371</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
border-radius: 20px;
border-color : white;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(72, 72, 72, 255), stop:1 rgba(90, 90, 90, 255));
color: white;
}</string>
   </property>
  </widget>
  <widget class="QComboBox" name="author_list">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>220</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(72, 72, 72, 255), stop:1 rgba(90, 90, 90, 255));
border-style: offset;
}</string>
   </property>
  </widget>
  <widget class="QComboBox" name="category_list">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>220</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(72, 72, 72, 255), stop:1 rgba(90, 90, 90, 255));
border-style: offset;
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_new_author">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>260</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>false</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: rgba(255, 255, 255, 92);
border-style:offset;
color:white;
}</string>
   </property>
   <property name="text">
    <string>Add new author</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_new_category">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>260</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>false</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: rgba(255, 255, 255, 92);
border-style:offset;
color:white;
}</string>
   </property>
   <property name="text">
    <string>Add new category</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="add_author">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(72, 72, 72, 255), stop:1 rgba(90, 90, 90, 255));
border-style: offset;
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="add_category">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>260</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(72, 72, 72, 255), stop:1 rgba(90, 90, 90, 255));
border-style: offset;
}</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
